lib/elixir/lib/enum.ex

```diff
@@ -764,15 +775,15 @@ defmodule Enum do
   end
 
   defp fetch_enumerable(enumerable, index, module) do
-    reduce_result =
+    reduce_res =
```

Comment:
it was done after these changes to the code I previously submitted.
- https://github.com/elixir-lang/elixir/commit/4b3b4370d8b821d92a368d5f58c1247044092473#diff-6881431a92cd4e3ea0de82bf2338f8eaL754
- https://github.com/elixir-lang/elixir/commit/4b3b4370d8b821d92a368d5f58c1247044092473#diff-6881431a92cd4e3ea0de82bf2338f8eaL773